docs/build.sbt

```diff
@@ -15,6 +15,18 @@ lazy val docs = project
   )
   .settings(PlayEbean.unscopedSettings: _*)
   .settings(inConfig(Test)(Seq(
+    playEbeanQueryGenerate := false,
```

Comment:
I think I'd name these just "ebean..." instead of "playEbean...".  It's much more concise that way. The query beans also don't have anything to do with Play really, so leaving Play out of the name would make it easier to refactor this into a more general sbt plugin in the future